src/style/app.module.css

```diff
@@ -2037,7 +2095,19 @@ input[type='radio']:checked + label:hover {
 }
 
 .inputFields {
+  background-color: var(--white-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`